lib/actions/projectOverviewResponse.js

```diff
@@ -6,7 +6,7 @@ class ProjectOverviewResponse {
   }
 
   sendToBot(bot, message) {
-    const listLinks = this.projectLinks.map((projectLink, i) => `*  *${projectLink.label}* ${projectLink.url}`).join('\n');
+    const listLinks = this.projectLinks.map(projectLink => `*  *${projectLink.label}* ${projectLink.url}`).join('\n');
```

Comment:
I'm guessing this was a linting error that you fixed. If this comes up in the future, I recommend handling unrelated fixes in a separate commit. It's totally OK by me to fix this kind of stuff in an unrelated PR, but if you keep the commit separate, it's easier to track what's going on in the commit history and if you have to roll something back, you don't have 2 unrelated things in a single commit that you might have to rollback together, when you really only want to rollback one of them.

Reply:
got it. I have to keep the consistency going and not be lazy on my commits.